### ISSUE TITLE
saml2-js potential security Vulnerability

### ISSUE THREAD

**Issue by AsafEini** [Clever/saml2#209]
since couple of days ago a security vulnerability was detected at a library forge-node.

saml2-js requires xml-encryption which requires node-forge.

should i be worry?

https://snyk.io/vuln/SNYK-JS-NODEFORGE-598677

**Comment by deanefrati** [comment]
Our scans show this as well. The issues is fixed in node-forge 0.10.0. Are there any plans to update this dependency and if so when do you expect to do so? This has a high CVSS score.

**Comment by bertyhell** [comment]
There is a PR that fixes this: https://github.com/Clever/saml2/pull/211

For now you can work around the security issue using npm-force-resolutions

Add this these in your package json:
```
{
  "scripts": {
    "preinstall": "npx npm-force-resolutions",
  },
  "resolutions": {
    "xml-encryption": "^1.2.1"
  }
}
```

Then install dependencies again:
```
npm i
```

**Comment by mcab** [comment]
This particular library was bumped with #211, and should be fixed in 2.0.6. Thanks for the report!
